lib/src/components/NxTransferListHalf/NxTransferListHalf.tsx

```diff
@@ -69,14 +69,14 @@ function TransferListItem<T extends string | number = string>(props: TransferLis
             <NxButton type="button"
                       variant="icon-only"
                       title={moveUpButtonTitle}
-                      disabled={isFilteredItem || isTopItem}
+                      className={isFilteredItem || isTopItem ? 'disabled' : ''}
```

Comment:
Updated `TransferListItem`'s `NxButton`s such that it now uses the `disabled` class instead of the `disabled` attribute. As such, tooltips are now enabled for its disabled buttons.

Reply:
> Consequently, native tooltips are now enabled for its disabled buttons.

We don't want native tooltips on anything.

Reply:
> We don't want native tooltips on anything.

Just to confirm. Should `TransferListItem`'s disabled "Move Up", "Move Down" buttons have MUI tooltips instead of native ones? Or should those disabled buttons have no tooltip at all?

Reply:
> The rules for tooltips and disabling should be as follows:
> 
> When not disabled at all:
> 
> * When `title` is provided to `NxButton`, the button gets an MUI tooltip
> * When `title` is not provided, the button gets no tooltip
> 
> When disabled via `prop`:
> 
> * When `title` is provided, throw an error
> * When `title` is not provided, the button gets no tooltip
> * The button's `onClick` handler does not get called
> 
> When disabled via class:
> 
> * When `title` is provided to `NxButton`, the button gets an MUI tooltip
> * When `title` is not provided, the button gets no tooltip
> * The button's `onClick` handler _does_ get called when the button is activated. It is up to the handler to do nothing in this case.  Whether this could/should be improved or adjusted could be debated, but this is how it is now and changing it is out of scope.
> 
> That the transfer list ordering buttons are currently misbehaving (on `main` too, not just this branch) is also out of scope, it is a bug relating to the logic around nested `NxTooltip`s. I'm not certain whether that should be fixed in `NxTransferListHalf` or in `NxButton` or even `NxTooltip` itself. But figuring that out should be a separate effort and ticket.

Through Slack, Ross and I agreed that the disabled buttons should have tooltips. So the current solution using the `disabled` class stays. The disabled buttons will use native tooltips for now, but they will be updated to MUI tooltips in a future update. I'm going to consider this conversation as resolved.